src/System.Private.CoreLib/shared/System/Text/DecoderFallback.cs

```diff
@@ -10,15 +11,15 @@ namespace System.Text
 {
     public abstract class DecoderFallback
     {
-        private static DecoderFallback s_replacementFallback; // Default fallback, uses no best fit & "?"
-        private static DecoderFallback s_exceptionFallback;
+        private static DecoderFallback? s_replacementFallback; // Default fallback, uses no best fit & "?"
+        private static DecoderFallback? s_exceptionFallback;
 
         public static DecoderFallback ReplacementFallback =>
-            s_replacementFallback ?? Interlocked.CompareExchange(ref s_replacementFallback, new DecoderReplacementFallback(), null) ?? s_replacementFallback;        
+            s_replacementFallback ?? Interlocked.CompareExchange(ref s_replacementFallback, new DecoderReplacementFallback(), null) ?? s_replacementFallback!;        
 
 
         public static DecoderFallback ExceptionFallback =>
-            s_exceptionFallback ?? Interlocked.CompareExchange<DecoderFallback>(ref s_exceptionFallback, new DecoderExceptionFallback(), null) ?? s_exceptionFallback;
+            s_exceptionFallback ?? Interlocked.CompareExchange<DecoderFallback?>(ref s_exceptionFallback, new DecoderExceptionFallback(), null) ?? s_exceptionFallback!;
```

Comment:
These are both places where a similar comment would be helpful.  I'll stop commenting on them. Thanks.